packages/fluent-ui/src/RangeWidget/RangeWidget.tsx

```diff
@@ -43,8 +43,8 @@ const RangeWidget = ({
   value,
   readonly,
   disabled,
-  onBlur,
-  onFocus,
+  // onBlur,
+  // onFocus,
```

Comment:
Delete this maybe since it is not used... or perhaps you meant to uncomment?

Reply:
```suggestion
```